src/SUMMARY.md

```diff
@@ -97,6 +97,10 @@
     * [Search Aggregates](datasets/search/search_aggregates/reference.md)
     * [Search Clients Daily](datasets/search/search_clients_daily/reference.md)
     * [Search Clients Last Seen](datasets/search/search_clients_last_seen/reference.md)
+  * [NonDesktop Datasets](datasets/non_desktop.md)
```

Comment:
I went back and forth on this as well. I opted for Non Desktop because the products aren't exclusively just the mobile browsers. For Day 2-7 echo tv and lockwise are included. For the gplay and app store there's vpn, lockwise and webvr in there. So I figured Non Desktop better reflects all of that. Maybe we should define mobile as being specific to mobile browsers. I think people use them interchangeably and they really shouldn't be.

Reply:
I went back and forth on this as well. I opted for Non Desktop because the products aren't exclusively just the mobile browsers. For Day 2-7 echo tv and lockwise are included. For the gplay and app store there's vpn, lockwise and webvr in there. So I figured Non Desktop better reflects all of that with mobile being a subset. Maybe we should define mobile as being specific to mobile browsers. I think people use them interchangeably and I don't think we should.